readout.sh

```diff
@@ -66,33 +67,9 @@ make ${JOBS+-j $JOBS} install
 
 #ModuleFile
 mkdir -p etc/modulefiles
-cat > etc/modulefiles/$PKGNAME <<EoF
-#%Module1.0
-proc ModulesHelp { } {
-  global version
-  puts stderr "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
-} 
-set version $PKGVERSION-@@PKGREVISION@$PKGHASH@@
-module-whatis "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
-# Dependencies
-module load BASE/1.0                                                          \\
-            ${BOOST_REVISION:+boost/$BOOST_VERSION-$BOOST_REVISION}            \\
-            ${GCC_TOOLCHAIN_REVISION:+GCC-Toolchain/$GCC_TOOLCHAIN_VERSION-$GCC_TOOLCHAIN_REVISION} \\
-            Monitoring/$MONITORING_VERSION-$MONITORING_REVISION               \\
-            Configuration/$CONFIGURATION_VERSION-$CONFIGURATION_REVISION      \\
-            Common-O2/$COMMON_O2_VERSION-$COMMON_O2_REVISION                  \\
-            libInfoLogger/$LIBINFOLOGGER_VERSION-$LIBINFOLOGGER_REVISION      \\
-            ReadoutCard/$READOUTCARD_VERSION-$READOUTCARD_REVISION            \\
-            ${LZ4_REVISION:+lz4/$LZ4_VERSION-$LZ4_REVISION}                   \\
-            FairLogger/$FAIRLOGGER_VERSION-$FAIRLOGGER_REVISION               \\
-            FairMQ/$FAIRMQ_VERSION-$FAIRMQ_REVISION                           \\
-            ${ZEROMQ_REVISION:+ZeroMQ/$ZEROMQ_VERSION-$ZEROMQ_REVISION}       \\
-            Control-OCCPlugin/$CONTROL_OCCPLUGIN_VERSION-$CONTROL_OCCPLUGIN_REVISION
-
+alibuild-generate-module --bin --lib > etc/modulefiles/$PKGNAME
+cat >> etc/modulefiles/$PKGNAME <<EoF
 # Our environment
 set READOUT_ROOT \$::env(BASEDIR)/$PKGNAME/\$version
```

Comment:
Not needed, I guess, since it's in any case a local variable no one uses. Or you actually meant **setenv**?

Reply:
Hello @ktf
I left it, as it was there before, and saw it done in the same way elsewhere, eg in the monitoring recipe. But indeed, if there's no purpose, it can be removed.